joint-discovery-gatk4.wdl

```diff
@@ -534,11 +539,11 @@ task IndelsVariantRecalibrator {
       -resource dbsnp,known=true,training=false,truth=false,prior=2:${dbsnp_resource_vcf}
   }
   runtime {
-    docker: docker_image
-    memory: mem_size
+    docker: docker
+    memory: "26 GB"
```

Comment:
Holy moly, really???  These parameters must have come from the 20K WGS WDL.  More typical cohort sizes won't need this much.

Reply:
I actually can't think of a good way to look this up.  Zamboni started all the GATK tasks at 4GB and retried until it succeeded or hit the max.  I found a 1000 exome callset that needed 16GB, so maybe this isn't so far off.  Unlike a lot of tools, VariantRecalibrator holds all the variants in memory